projects/Amlogic/packages/aml-s9xx-device-trees/package.mk

```diff
@@ -0,0 +1,49 @@
+################################################################################
+#      This file is part of LibreELEC - http://www.libreelec.tv
+#      Copyright (C) 2016 Mateusz Krzak (kszaquitto@gmail.com)
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="aml-s9xx-device-trees"
+PKG_VERSION="097d287"
+PKG_LICENSE="OSS"
+PKG_URL="https://github.com/kszaq/s905-device-trees/archive/$PKG_VERSION.tar.gz"
```

Comment:
shouldn't we fork that under our account ?
`https://github.com/LibreELEC/s905-device-trees/archive/$PKG_VERSION.tar.gz`
@chewitt @lrusak ?

Reply:
I'm fine with it as-is for now, but long term we should have the project host something as our experience shows people contribute more to a project repo than a personal one.

Reply:
NB: the same applies to a few other things among recent Amlogic changes, but I see them as an admin/housekeeping task to be done as we approach beta release.

Reply:
There has been a number of community contributions since the fd628-aml merge, it seems to have perked peoples interest and I've had 3 dtbs submitted to me in the last 24 hours so I would be in favour of a fork for people to contribute to.

Reply:
https://github.com/LibreELEC/device-trees-amlogic is open for business .. done as a root fork; not to disrespect where things started but simply to ensure we are the obvious upstream/root target for things.